### PR TITLE
Add terms from dev-es branch to main

### PR DESCRIPTION
**Pull request by krol3**
### Describe your changes

Merge the `dev-es` branch to the main

### Related issue number or link (ex: `resolves #issue-number`)
N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [x ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by electrocucaracha — APPROVED** [review]

File: content/es/_index.md [hunk 1]
```diff
@@ -5,25 +5,43 @@ status: Completed
...
+Asegúrese de seguir la [Guía de estilo](/es/style-guide/), leer el documento de [Cómo contribuir](/es/contribute/) y unirse al canal de [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) en el CNCF Slack.
+También hay un canal [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) para aquellos que quieran ayudar a traducir el glosario a su idioma.
 
 ## Reconocimientos
 
-El Glosario Cloud Native fue iniciado por CNCF Marketing
-Committee (Business Value Subcommittee) e incluye
-contribuciones de [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
-[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) y muchos contribuidores mas. Para una lista de contribuidores completa, consulte [esta página de GitHub](https://github.com/cncf/glossary/graphs/contributors).
```
Comment (seokho-son) [inline content/es/_index.md:23]: Hi @krol3 
I think this part from the source document has been removed by this PR. (Acknowledgement to the Glossary maintainers)
But it seems not urgent issue, can be resolved next time.
I thinks you can open another PR to restore the removed part to make up this PR, or I can merge this PR first, upto your team decision.

**Review by seokho-son — APPROVED** [review]
LGTM with a minor comment. :)

**Review by ramrodo — APPROVED** [review]
/lgtm

File: content/es/_index.md (hunk 1 above)
Comment (ramrodo) [reply content/es/_index.md:23]: I agree with opening other PR to remove this. I think we can merge this PR first.

**Comment by seokho-son** [comment]
Let me merge this PR. Thanks all !!

File: content/es/_index.md (hunk 1 above)
Comment (krol3) [reply content/es/_index.md:23]: I created an issue to track this!! Thank you

**Pull request merged**